tests/acceptance/index-test.js

```diff
@@ -268,8 +268,10 @@ describe('Acceptance | index', function() {
     await visit('/')
     expect(find('[data-test-review-warning]')).to.have.length(1)
     await click('[data-test-review-warning]')
-    expect(currentURL()).to.equal(`
+
+    let expectedUrl = `
```

Comment:
If you're not mutating the variable, we usually prefer to use `const` over `let`.